src/unit-tests/osloader-test/CMakeLists.txt

```diff
@@ -15,8 +17,8 @@ while(MOD GREATER 0)
   set_target_properties(MODULE${MOD} PROPERTIES 
     COMPILE_DEFINITIONS "MODULE_NAME=module${MOD}" 
     PREFIX ""
-    LIBRARY_OUTPUT_DIRECTORY eeprom1)
+    # TODO-MAC:
+    # Forcing eeprom1 to be in CMAKE_BINARY_DIR.
+    LIBRARY_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR}/eeprom1)
+  target_link_libraries(osal_loader_UT MODULE${MOD})
```

Comment:
This specific line can be extracted to a separate PR also with the `add_osal_ut_exe` line moved above like on the diff. I think I can also do it rather soon.